hardhat.config.ts

```diff
@@ -28,8 +28,28 @@ const config: HardhatUserConfig = {
                         runs: 200
                     }
                 }
+            },
+            {
+                version: "0.8.8",
+                settings: {
+                    optimizer: {
+                        enabled: true,
+                        runs: 200
+                    }
+                }
             }
-        ]
+        ],
+        overrides: {
```

Comment:
I don't think override is required for individual files. Hardhat picks the target solc version from the pragma statement and compiles the contract accordingly. No harm in having this though.

Reply:
Yep you're right. The `overrides` configuration is only required if a) the pragma statement for a file allows a file to be compiled with a range of solc versions b) > 1 solc version specified under `compilers` could be used for the file and c) you want to use a specific solc version that is not highest version listed under `compilers`.

I ended up dropping the `overrides` configuration in the interest of minimizing config clutter.